### DISCUSSION TITLE
v3.6.0

### DISCUSSION THREAD

**Discussion by InfinityLoop1308** [InfinityLoop1308/PipePipe#335] [Announcements]
### New Features

- Support ReturnYouTubeDislike (YouTube)
- Update notification icon

### Bug Fixes

- Playback error occurs randomly (YouTube)
- Some spaces are replaced to &nbsp; in comments (YouTube)
- Unable to parse URLs shared from official client with timestamp (BiliBili)
- Unable to fetch comments of some videos(BiliBili)

<hr/>
If you're enjoying PipePipe, please help share it with your friends. Every bit of support means a lot to us!
<hr/>

### Which APK should you download?

- arm64-v8a should work on most devices.
- armeabi-v7a if your device is a 10+ year old phone.
- x86_64 for 64-bit Android simulators (on a PC).
- x86 for 32-bit simulators.

If these rules not working, download the universal APK. It should work on any device.
All APKs provide the same contents and functions, but the universal one may be slower than the split ones.

<hr /><em>This discussion was created from the release <a href='https://github.com/InfinityLoop1308/PipePipe/releases/tag/'>v3.6.0</a>.</em>

**Comment by jojojojojoj5564656465465** [comment]
Is it a way to export or import my settings?

**Reply by InfinityLoop1308** [reply]
Settings -> Content -> Import Database

**Comment by Alexia99-lab** [comment]
Ugh..video can't play and bad loading🙁

**Reply by InfinityLoop1308** [reply]
Check 3.6.1
